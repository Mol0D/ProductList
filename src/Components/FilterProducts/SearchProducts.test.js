import React from 'react';
import {shallow, mount} from 'enzyme';
import SearchProduct from './SearchProducts'; 

describe("SearchProducts", () =>{
    const props ={
        data: [{}, {}, {}]
    }

    it("without component", () =>{
        const wrapper = shallow(<SearchProduct/>);
        expect(wrapper).toMatchSnapshot();
    }) 

    it("with component", () =>{
        const wrapper = shallow(<SearchProduct {...props}/>)
        expect(wrapper).toMatchSnapshot();
    })

    it("correct Prop Data", () =>{
        const wrapper = mount(<SearchProduct data={props.data}/>);
        expect(wrapper.prop('data')).toEqual([{}, {}, {}])
    })
})

