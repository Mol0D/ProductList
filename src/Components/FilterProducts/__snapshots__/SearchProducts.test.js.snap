// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchProducts with component 1`] = `ShallowWrapper {}`;

exports[`SearchProducts without component 1`] = `ShallowWrapper {}`;
